src/classes/loggers/issue-logger.ts

```diff
@@ -70,10 +69,12 @@ export class IssueLogger extends Logger {
   }
 
   private _getIssuePrefix(): string {
-    return chalk.red(`[#${this._getIssueNumber()}]`);
+    // return chalk.red(`[#${this._getIssueNumber()}]`);
```

Comment:
Why the comment? WIP?

Reply:
Well, my tests were failing cause the colored text was different (?) than the one passed to `.toHaveBeenCalledWith(..)`. Is it just me? That was the main motivation to extract the 'coloring' logic into a decorator class. So that my logger test suite wouldn't break... Since these methods are specific to `IssueLogger`, I didn't want to introduce another `StylingIssueLoggerDecaorator` just for that, so I left them out for now.